openapi.yaml

```diff
@@ -12561,14 +12560,15 @@ paths:
       servers:
       - url: https://api.linode.com/v4
       summary: Object Storage Bucket Remove
-      description: >
-        Removes a single bucket. While buckets containing objects _may_ be
-        deleted by including the `force` option in the request, such operations
-        will fail if the bucket contains too many objects. The recommended
+      description: |
+        Removes a single bucket.
+
+        Bucket objects must be removed prior to removing the bucket. While buckets containing objects _may_ be
```

Comment:
Do bucket objects need to be removed first? I thought I saw some chatter somewhere in Slack that indicated you did not need to remove first. You should confirm.

Reply:
You're correct @yoshizzle